src/components/autocomplete-select/README.md

```diff
@@ -0,0 +1,26 @@
+# Auto-complete Select
+
+A select control (dropdown) with auto-complete functionality. This is a wrapper component for [react-select](https://github.com/JedWatson/react-select).
+
+## Usage
+
+```jsx
+import { AutocompleteSelect } from '@nomios/web-uikit';
+
+const myOptions = [
+    { value: 'car', label: 'Car' },
+    { value: 'bike', label: 'Bike' },
+    { value: 'truck', label: 'Truck' },
+];
+
+<AutocompleteSelect options={ myOptions } />
+```
+
+## Props
+
+| name | type | default | description |
+| ---- | ---- | ------- | ----------- |
+| className | string | | A classname to override styles. This classname will be applied to the `SelectContainer`. |
```

Comment:
Maybe we can remove `className` from here as it's already supported by `react-select`? This means we can remove the whole table.

Reply:
Yep. The only reason I've mentioned the `className` here is to explain that it will be applied to the `SelectContainer`. But this info can be found in their documentation. And that way we can remove our table.